spec/granite/integrators/find_or_spec.cr

```diff
@@ -0,0 +1,26 @@
+require "../../spec_helper"
+
+describe "find_or_create_by, find_or_initialize_by" do
+  it "creates on find_or_create when not found" do
+    Parent.clear
+    Parent.find_or_create_by(name: "name")
+    Parent.all.first.name.should eq("name")
+    Parent.all.first.new_record?.should eq(false)
```

Comment:
Super minor but `Parent.all.first` could be shortened a bit to `Parent.first!`.

Reply:
fixed in latest